errors.ts

```diff
@@ -0,0 +1,20 @@
+export enum ErrorKind {
```

Comment:
[cbb] I've been staying away from TS enums. They have a lot of issues.

Reply:
What should I do instead?

Reply:
```ts
type ErrorKind = "Unauthorized" | "UserError";
```

https://w3ctag.github.io/design-principles/#string-constants